src/features/Dashboard/AssignedClasses/ClassCard.jsx

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { Link } from 'react-router-dom';
 import { Col } from '@edx/paragon';
 
 import { formatDateRange } from 'react-paragon-topaz';
 
-const ClassCard = ({ data }) => (
-  <Col sm={12} md={4} className="class-card-container px-4 py-3 ">
-    <h4>{data?.className}</h4>
-    <p className="course-name">{data?.masterCourseName}</p>
-    <p className="date"><i className="fa-sharp fa-regular fa-calendar-day" />{formatDateRange(data.startDate, data?.endDate)}</p>
-  </Col>
-);
+import { useInstitutionIdQueryParam } from 'hooks';
+
+const ClassCard = ({ data }) => {
+  const addQueryParam = useInstitutionIdQueryParam();
+  const url = addQueryParam(`/classes/${data?.classId}`);
+
+  return (
+    <Col sm={12} md={4} className="class-card-container px-4 py-3 ">
+      <Link className="h4" to={url}>{data?.className}</Link>
```

Comment:
Change the color text to indicate is a link and add the previous param as home page